addons/crm_claim/migrations/8.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,27 @@
+<openerp>
+  <data>
+    <record id="stage_claim1" model="crm.claim.stage"><field name="state"/>
+        <field name="sequence">1</field>
+    </record>
+    <record id="stage_claim3" model="crm.claim.stage">
+      <field name="fold"/>
+      <field name="case_refused"/>
+      <field name="state"/>
+    </record>
+    <record id="stage_claim2" model="crm.claim.stage">
+      <field name="state"/>
+    </record>
```

Comment:
state, fold, case_refused fields are actually obsolete fields, for which the values were removed from the noupdate data. Sorry the compare script is not that smart! I would remove them, because loading these entries will give warnings about unknown fields being written to these stages.